.gitlab-ci.yml

```diff
@@ -1074,15 +1095,10 @@ test_full_integration:
     - tar -xf /tmp/workspace.tar.gz
     - mv /tmp/build_revisions.env /tmp/stage-artifacts .
 
-    # Post job status
-    - apk --update add curl jq
-    - ${CI_PROJECT_DIR}/scripts/github_pull_request_status pending "Gitlab ${CI_JOB_NAME} started" "${CI_JOB_URL}" "${CI_JOB_NAME}/${INTEGRATION_REV}"
```

Comment:
I think this has to remain where it is, because it needs to be executed after `workspace.tar.gz` is extracted. It may seem like it works in mender-qa, because Gitlab clones it, but if you launch a PR from say `integration`, it won't work because that repository is inside the `workspace.tar.gz` file.